tests/integration/test_login_email.js

```diff
@@ -0,0 +1,28 @@
+var system = require('system'),
+    test_username = system.env.TEST_USERNAME,
+    test_password = system.env.TEST_PASSWORD,
+    host = "http://localhost:7500";
+
+casper.test.begin('Lastuser redirects to correct Lastuser login page', 1, function (test){
+    casper.start(host, function() {
+        this.echo("Check for running server");
+        test.assertHttpStatus(200, '200 OK');
+        casper.clear();
+        phantom.clearCookies();
+        casper.thenOpenAndEvaluate(host+"/login", function(test_username, test_password){
+            document.querySelector('#username').value = username;
+            document.querySelector('#password').value = password;
```

Comment:
How are `username` and `password` in scope here? Shouldn't these be assigned to `test_username` and `test_password`?